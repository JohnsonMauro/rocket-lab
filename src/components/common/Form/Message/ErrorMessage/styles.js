import styled from '@emotion/styled';

const ErrorMessageStyles = styled.span`
  color: ${({ theme }) => theme.colors.error};
  font-weight: 600;
  display: block;
  font-size: 12px;
  padding-left: 5px;
`;

export { ErrorMessageStyles };
