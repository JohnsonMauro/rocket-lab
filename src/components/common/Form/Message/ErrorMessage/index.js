import { ErrorMessageStyles } from './styles';

const ErrorMessage = ({ children }) => <ErrorMessageStyles>{children}</ErrorMessageStyles>;

export default ErrorMessage;
