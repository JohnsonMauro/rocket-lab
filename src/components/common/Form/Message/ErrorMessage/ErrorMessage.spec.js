import React from 'react';
import { render } from '@testing-library/react';

import GlobalsContainer from '@test/GlobalsContainer';
import ErrorMessage from './index';

describe('<ErrorMessage />', () => {
  it('Renders the ErrorMessage component', () => {
    const { container } = render(<ErrorMessage>Invalid</ErrorMessage>, {
      wrapper: GlobalsContainer,
    });
    expect(container).toMatchSnapshot();
  });
});
