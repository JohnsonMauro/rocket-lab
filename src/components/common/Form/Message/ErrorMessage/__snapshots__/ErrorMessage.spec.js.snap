// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> Renders the ErrorMessage component 1`] = `
<div>
  <span
    class="css-13qry7b-ErrorMessageStyles e1gulyup0"
  >
    Invalid
  </span>
</div>
`;
